rekcurd_dashboard/README.md

```diff
@@ -1,19 +1,36 @@
 # Rekcurd dashboard backend
-## Edit settings.yml
-```
-$ vi settings.yml
-```
+You can use `setting.yml` as your configuration file. See [template](./template/settings.yml-tpl).
+
 
 ## Run it!
 ### DB
+##### help
 ```bash
-$ python app.py --settings settings.yml db init
-$ python app.py --settings settings.yml db migrate
+$ python app.py db -h
 ```
 
+##### initialization
+```bash
+$ python app.py db --settings settings.yml init
+$ python app.py db --settings settings.yml migrate
+```
+
+##### migration
+```bash
+$ python app.py db --settings settings.yml migrate
+$ python app.py db --settings settings.yml upgrade
+```
+
+
 ### Server
+##### help
+```bash
+$ python app.py server -h
+```
+
+##### boot
 ```bash
-$ python app.py --settings settings.yml server
+$ python app.py server --settings settings.yml
```

Comment:
these are `rekcurd_dashboard/core/create_app.py` , not `app.py` ?

Reply:
It was my mistake.
`rekcurd_dashboard server --settings....` is correct.